lib/modules/hosts/pornbot.js

```diff
@@ -0,0 +1,89 @@
+//Borrowed heavily from gfycat.js
+modules['showImages'].siteModules['pornbot'] = {
+	domains: [ 'pornbot.net' ],
+	detect: function(href, elem) {
+		return (/^https?:\/\/(?:w{3}\.)?(?:v\.)?pornbot\.net\/[a-z0-9]{8,}$/i).test(href);
+	},
+	handleLink: function(elem) {
+		var hashRe = /^https?:\/\/(?:w{3}\.)?(?:v\.)?pornbot\.net\/[a-z0-9]{8,}$/i,
+			def = $.Deferred(),
+			groups = hashRe.exec(elem.href);
+
+		if (groups) {
+			var apiURL = location.protocol + '//pornbot.net/ajax/info.php?v=' + encodeURIComponent(groups[2]);
+			elem.href = location.protocol + '//v.pornbot.net/' + encodeURIComponent(groups[2]);
```

Comment:
Chrome was throwing errors on https://v.pornbot.net/ because you're using the cloudfront certificate. Can you either get your own certificate or explicitly change `location.protocol` to `http:` ?